support-frontend/assets/components/payPalPaymentButton/payPalRecurringContainer.tsx

```diff
@@ -55,7 +63,13 @@ export function PayPalButtonRecurringContainer({
 		csrf,
 		isTestUser: isTestUser ?? false,
 		setValidationControls,
-		onClick: () => dispatch(sendFormSubmitEventForPayPalRecurring()),
+		onClick: () =>
+			trackCheckoutSubmitAttempt(
+				`PayPal-${contributionType}-submit`,
+				`npf-allowed-for-user-type-${userTypeFromIdentityResponse}`,
+				'PayPal',
+				isSupporterPlus ? 'SupporterPlus' : 'Contribution',
+			),
```

Comment:
This was the only thing the `sendFormSubmitEventForPayPalRecurring` action was actually doing as far as I can tell.